vendor.conf

```diff
@@ -0,0 +1,57 @@
+# go-runc client for runc; following master HEAD on 12/16/2016
+github.com/crosbymichael/go-runc 2fdd3cd06f7443a55f6eb14ead2cac507506e718
+# distribution code from Docker; v2.6.0-rc.1 tag
+github.com/docker/distribution v2.6.0-rc.1
+# go-metrics client to prometheus; following master HEAD on 12/16/2016
+github.com/docker/go-metrics 0f35294225552d968a13f9c5bc71a3fa44b2eb87
+# prometheus client; v0.8.0
+github.com/prometheus/client_golang v0.8.0
+# prometheus client model; following master HEAD on 12/16/2016
+github.com/prometheus/client_model fa8ad6fec33561be4280a8f0514318c79d7f6cb6
+# prometheus common library; following master HEAD on 12/16/2016
+github.com/prometheus/common 195bde7883f7c39ea62b0d92ab7359b5327065cb
+# prometheus procfs; following master HEAD 12/16/2016
+github.com/prometheus/procfs fcdb11ccb4389efb1b210b7ffb623ab71c5fdd60
+# beorn7/perks; following master HEAD 12/16/2016
+github.com/beorn7/perks 4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9
+# matttproud/golang_protobuf_extensions; v1.0.0 release
+github.com/matttproud/golang_protobuf_extensions v1.0.0
+# go-units from Docker; latest release = 0.3.1
+github.com/docker/go-units v0.3.1
+# gogo/protobuf - following master HEAD 12/16/2016 (latest Release doesn't have needed change)
+github.com/gogo/protobuf 06ec6c31ff1bac6ed4e205a547a3d72934813ef3
+# golang support for protobufs - follower master HEAD on 12/16/2016
+github.com/golang/protobuf 8ee79997227bf9b34611aee7946ae64735e6fd93
+# nats/go-nats, v1.2.2 release (Oct 2016)
+github.com/nats-io/go-nats v1.2.2
+github.com/nats-io/nats v1.2.2
+# nats nuid package; v1.0.0 release
+github.com/nats-io/nuid v1.0.0
+# nats-streaming-server, v0.3.4 release (Dec 2016)
+github.com/nats-io/nats-streaming-server v0.3.4
+github.com/nats-io/go-nats-streaming v0.3.4
+# gnatsd; v0.9.6 release
+github.com/nats-io/gnatsd v0.9.6
+# runc, latest release v1.0.0-rc2
+github.com/opencontainers/runc v1.0.0-rc2
+# OCI runtime spec, latest release v1.0.0-rc3
+github.com/opencontainers/runtime-spec v1.0.0-rc3
+# logrus, v0.11.0 latest release
+github.com/sirupsen/logrus v0.11.0
+# go-btrfs from stevvooe; following master HEAD on 12/16/2016
+github.com/stevvooe/go-btrfs a5487675780517f5c3e7dd697c70157466b8dcae
```

Comment:
Can you update `go-btrfs` to the current `HEAD`?  https://github.com/stevvooe/go-btrfs/pull/4 is now used in the tests.